From a193e732fa6fff2ce42001dcd7ff3b3694d69ed0 Mon Sep 17 00:00:00 2001
From: Daniel Landau <daniel@landau.fi>
Date: Sat, 11 Mar 2017 10:37:08 +0200
Subject: [PATCH] Use normal touch handling all the time

---
 drivers/input/touchscreen/it7260_ts_i2c.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/it7260_ts_i2c.c b/drivers/input/touchscreen/it7260_ts_i2c.c
index a45eb7137e0d..97cc2cb9d867 100644
--- a/drivers/input/touchscreen/it7260_ts_i2c.c
+++ b/drivers/input/touchscreen/it7260_ts_i2c.c
@@ -1619,8 +1619,10 @@ static irqreturn_t IT7260_ts_threaded_handler(int irq, void *devid)
 {
 	smp_rmb();
 	if (driverInLowPower) {
-		readTouchDataPoint_Ambient();
-		smp_wmb();
+    // Just always do normal touch point. _Ambient version doesn't work in Asteroid
+		readTouchDataPoint();
+		/* readTouchDataPoint_Ambient(); */
+		/* smp_wmb(); */
 		/* XXX: call readTouchDataPoint() here maybe ? */
 	} else {
 		readTouchDataPoint();
-- 
2.11.1

