From 672ee5e047434e723986b84d286157a366e90abb Mon Sep 17 00:00:00 2001
From: Philip Russell <argosphil@murena.io>
Date: Sun, 3 Sep 2023 16:11:58 +0000
Subject: [PATCH 2/3] remove tracepoints

---
 drivers/platform/msm/ipa/ipa_v2/ipa.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/platform/msm/ipa/ipa_v2/ipa.c b/drivers/platform/msm/ipa/ipa_v2/ipa.c
index a7c3e5f86e30..593904ef6fda 100644
--- a/drivers/platform/msm/ipa/ipa_v2/ipa.c
+++ b/drivers/platform/msm/ipa/ipa_v2/ipa.c
@@ -36,9 +36,6 @@
 #include "ipa_i.h"
 #include "../ipa_rm_i.h"
 
-#define CREATE_TRACE_POINTS
-#include "ipa_trace.h"
-
 #define IPA_SUMMING_THRESHOLD (0x10)
 #define IPA_PIPE_MEM_START_OFST (0x0)
 #define IPA_PIPE_MEM_SIZE (0x0)
-- 
2.41.0

