From adabcf465235c60caa3c87d25954afa2f9f2dab9 Mon Sep 17 00:00:00 2001
From: Arseniy <dodoradio@outlook.com>
Date: Tue, 6 Dec 2022 21:54:36 +0000
Subject: [PATCH] Fix compilation warnings

Upstream-Status: Inappropriate
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index c024749431a3..f9cd330600a9 100644
--- a/Makefile
+++ b/Makefile
@@ -401,9 +401,7 @@ KBUILD_CPPFLAGS := -D__KERNEL__
 
 KBUILD_CFLAGS   := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
 		   -fno-strict-aliasing -fno-common \
-		   -Werror-implicit-function-declaration \
 		   -Wno-format-security \
-		   -Werror \
 		   -std=gnu89
 
 KBUILD_AFLAGS_KERNEL :=
-- 
2.38.1

