From 16a863cd2af8e957d045baf3bc8d8d8f7a96298c Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Tue, 17 Oct 2017 19:21:51 +0200
Subject: [PATCH] Adapt to what tetra's hwcomposer expects

---
 hwcomposer/hwcomposer_backend_v11.cpp | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/hwcomposer/hwcomposer_backend_v11.cpp b/hwcomposer/hwcomposer_backend_v11.cpp
index 26ee1cc..d7c20bc 100644
--- a/hwcomposer/hwcomposer_backend_v11.cpp
+++ b/hwcomposer/hwcomposer_backend_v11.cpp
@@ -118,18 +118,18 @@ void HWComposer::present(HWComposerNativeWindowBuffer *buffer)
 {
     QPA_HWC_TIMING_SAMPLE(presentTime);
 
-    fblayer->handle = buffer->handle;
-    fblayer->releaseFenceFd = -1;
+    mlist[0]->hwLayers[0].handle = buffer->handle;
+    mlist[0]->hwLayers[0].releaseFenceFd = -1;
 
 #ifdef QPA_HWC_SYNC_BEFORE_SET
     int acqFd = getFenceBufferFd(buffer);
     if (acqFd >= 0) {
         sync_wait(acqFd, -1);
         close(acqFd);
-        fblayer->acquireFenceFd = -1;
+        mlist[0]->hwLayers[0].acquireFenceFd = -1;
     }
 #else
-    fblayer->acquireFenceFd = getFenceBufferFd(buffer);
+    mlist[0]->hwLayers[0].acquireFenceFd = getFenceBufferFd(buffer);
 #endif
 
     QPA_HWC_TIMING_SAMPLE(syncTime);
@@ -144,7 +144,7 @@ void HWComposer::present(HWComposerNativeWindowBuffer *buffer)
 
     QPA_HWC_TIMING_SAMPLE(setTime);
 
-    setFenceBufferFd(buffer, fblayer->releaseFenceFd);
+    setFenceBufferFd(buffer, mlist[0]->hwLayers[0].releaseFenceFd);
 
     if (mlist[0]->retireFenceFd != -1) {
         close(mlist[0]->retireFenceFd);
@@ -291,7 +291,7 @@ HwComposerBackend_v11::createWindow(int width, int height)
 
 
     HWComposer *hwc_win = new HWComposer(width, height, HAL_PIXEL_FORMAT_RGBA_8888,
-                                         hwc_device, hwc_mList, &hwc_list->hwLayers[1], num_displays);
+                                         hwc_device, hwc_mList, &hwc_list->hwLayers[0], num_displays);
     return (EGLNativeWindowType) static_cast<ANativeWindow *>(hwc_win);
 }
 
@@ -345,7 +345,7 @@ HwComposerBackend_v11::sleepDisplay(bool sleep)
     } else {
 #ifdef HWC_DEVICE_API_VERSION_1_4
         if (hwc_version == HWC_DEVICE_API_VERSION_1_4) {
-            HWC_PLUGIN_EXPECT_ZERO(hwc_device->setPowerMode(hwc_device, 0, HWC_POWER_MODE_NORMAL));
+//            HWC_PLUGIN_EXPECT_ZERO(hwc_device->setPowerMode(hwc_device, 0, HWC_POWER_MODE_NORMAL));
         } else
 #endif
 #ifdef HWC_DEVICE_API_VERSION_1_5
-- 
2.7.4

