From 4c2337a555c07020771ca3aa72304a269b400dc6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Darrel=20Gri=C3=ABt?= <dgriet@gmail.com>
Date: Wed, 17 Aug 2022 21:50:14 +0200
Subject: [PATCH] HybrisHrmAdaptor: Set default delay to 200 ms.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 adaptors/hybrishrmadaptor/hybrishrmadaptor.cpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/adaptors/hybrishrmadaptor/hybrishrmadaptor.cpp b/adaptors/hybrishrmadaptor/hybrishrmadaptor.cpp
index b1a7909..fd25bbc 100644
--- a/adaptors/hybrishrmadaptor/hybrishrmadaptor.cpp
+++ b/adaptors/hybrishrmadaptor/hybrishrmadaptor.cpp
@@ -42,9 +42,6 @@ HybrisHrmAdaptor::HybrisHrmAdaptor(const QString& id) :
     	sensordLogW() << "Path does not exists: " << powerStatePath;
     	powerStatePath.clear();
     }
-    // Set default delay.
-    unsigned int interval_us = 200 * 1000;
-    setInterval(0, interval_us);
 }
 
 HybrisHrmAdaptor::~HybrisHrmAdaptor()
@@ -54,6 +51,8 @@ HybrisHrmAdaptor::~HybrisHrmAdaptor()
 
 bool HybrisHrmAdaptor::startSensor()
 {
+    unsigned int interval_us = 200 * 1000;
+    setDefaultInterval(interval_us);
     if (!(HybrisAdaptor::startSensor()))
         return false;
     if (isRunning() && !powerStatePath.isEmpty())
-- 
2.49.0

