From d89b12e87dfc9fe5e7dd0c9231f3706a7f858152 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Wed, 13 Jul 2016 19:19:31 +0200
Subject: [PATCH] Don't use asus_chg_set_chg_mode without CONFIG_CHARGER_ASUS

---
 drivers/usb/phy/phy-msm-usb.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/phy/phy-msm-usb.c b/drivers/usb/phy/phy-msm-usb.c
index 684d8ff..9b7816e 100755
--- a/drivers/usb/phy/phy-msm-usb.c
+++ b/drivers/usb/phy/phy-msm-usb.c
@@ -2947,7 +2947,9 @@ static void msm_otg_sm_work(struct work_struct *w)
 			msm_otg_start_peripheral(otg, 0);
 			motg->chg_type = USB_DCP_CHARGER;
 			clear_bit(B_FALSE_SDP, &motg->inputs);
+#ifdef CONFIG_CHARGER_ASUS
 			asus_chg_set_chg_mode(ASUS_CHG_SRC_DC);
+#endif
 			printk("[USB] set_chg_mode: ASUS DCP\n");
 			otg->phy->state = OTG_STATE_B_IDLE;
 			work = 1;
-- 
2.8.1