From 6470c907014f3f892b8f443fdeceec7663afb0c7 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sat, 8 Jul 2017 14:12:06 +0100
Subject: [PATCH] Rotate screen for harmony

---
 src/compositor/lipstickcompositor.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/compositor/lipstickcompositor.cpp b/src/compositor/lipstickcompositor.cpp
index 4dce9d7b..174c5519 100644
--- a/src/compositor/lipstickcompositor.cpp
+++ b/src/compositor/lipstickcompositor.cpp
@@ -376,6 +376,8 @@ void LipstickCompositor::onSurfaceDying()
         item->m_windowClosed = true;
         item->tryRemove();
     }
+
+    setScreenOrientation(Qt::InvertedLandscapeOrientation);
 }
 
 void LipstickCompositor::initialize()
