From 62ebb71c2c620ef0de44cd16ab1d7c4aee96f2c1 Mon Sep 17 00:00:00 2001
From: MagneFire <IDaNLContact@gmail.com>
Date: Mon, 26 Oct 2020 21:20:24 +0100
Subject: [PATCH] HybrisAlsAdaptor: Set default delay. Fixes excessive lags on
 tetra.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate
Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp b/adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp
index b02decb..8765cd5 100644
--- a/adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp
+++ b/adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp
@@ -40,6 +40,9 @@ HybrisAlsAdaptor::HybrisAlsAdaptor(const QString& id) :
         qCWarning(lcSensorFw) << NodeBase::id() << "Path does not exists: " << powerStatePath;
     	powerStatePath.clear();
     }
+    // Set default delay.
+    unsigned int interval_us = 200 * 1000;
+    setInterval(0, interval_us);
 }
 
 HybrisAlsAdaptor::~HybrisAlsAdaptor()
-- 
2.49.0

