From ebc7675c6e3205798e087e3e602041f575bcd4c4 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Wed, 26 Jul 2017 22:27:37 +0100
Subject: [PATCH 2/2] GestureFilterArea: Fix swipe interaction on rotated
 screens

---
 gesturefilterarea.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gesturefilterarea.cpp b/gesturefilterarea.cpp
index 6436aab..eb0b6fe 100644
--- a/gesturefilterarea.cpp
+++ b/gesturefilterarea.cpp
@@ -85,8 +85,8 @@ void GestureFilterArea::mouseMoveEvent(QMouseEvent *event) {
     }
     m_counter++;
 
-    m_velocityX = (m_velocityX*(m_counter-1) + (event->windowPos().x()-m_prevPos.x()))/m_counter;
-    m_velocityY = (m_velocityY*(m_counter-1) + (event->windowPos().y()-m_prevPos.y()))/m_counter;
+    m_velocityX = (m_velocityX*(m_counter-1) + -((event->windowPos().x()-m_prevPos.x()))/m_counter);
+    m_velocityY = (m_velocityY*(m_counter-1) + -((event->windowPos().y()-m_prevPos.y()))/m_counter);
     if(m_tracing) {
         if(m_velocityX > 4) {
             m_tracing = false;
@@ -128,7 +128,7 @@ void GestureFilterArea::mouseMoveEvent(QMouseEvent *event) {
         else
             delta = event->windowPos().y() - m_prevPos.y();
 
-        emit swipeMoved(m_horizontal, delta);
+        emit swipeMoved(m_horizontal, -delta);
     }
     m_prevPos = event->windowPos();
 }
-- 
2.7.4

